cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -235,22 +263,43 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
 
     sampleAllIndicesAndWait()
 
-    profileQuery(title = "Node index range seek",
+    profileQuery(title = "Node Index Seek By Range",
                  text =
-                   """Finds nodes using an index seek where the value of the property matches a given prefix string.
-                     |This operator can be used for `STARTS WITH` and comparators such as `<`, `>`, `\<=` and `>=`""".stripMargin,
+                   """The `NodeIndexSeekByRange` operator finds nodes using an index seek where the value of the property matches a given prefix string.
+                     |`NodeIndexSeekByRange` can be used for `STARTS WITH` and comparison operators such as `<`, `>`, `\<=` and `>=`.
+                     |If the index is a unique index, the operator is instead called `NodeUniqueIndexSeekByRange`.""".stripMargin,
                  queryText = "MATCH (l:Location) WHERE l.name STARTS WITH 'Lon' RETURN l",
-                 assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString(IndexSeekByRange.name))
+                 assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("NodeIndexSeekByRange"))
+    )
+  }
+
+  @Test def nodeUniqueIndexRangeSeek() {
+    executePreparationQueries {
+      val a = (0 to 100).map { i => "CREATE (:Team)" }.toList
+      val b = (0 to 300).map { i => s"CREATE (:Team {name: '$i'})" }.toList
+      a ++ b
+    }
+
+    sampleAllIndicesAndWait()
+
+    profileQuery(title = "Node Unique Index Seek By Range",
+      text =
+        """The `NodeUniqueIndexSeekByRange` operator finds nodes using an index seek within a unique index, where the value of the property matches a given prefix string.
+          |`NodeUniqueIndexSeekByRange` can be used for `STARTS WITH` and comparison operators such as `<`, `>`, `\<=` and `>=`.
+          |If the index is not unique, the operator is instead called `NodeIndexSeekByRange`.""".stripMargin,
+      queryText = "MATCH (t:Team) WHERE t.name STARTS WITH 'Ma' RETURN t",
+      assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("NodeUniqueIndexSeekByRange"))
     )
   }
 
+
   @Test def nodeIndexScan() {
     executePreparationQueries((0 to 250).map { i =>
       "CREATE (:Location)"
     }.toList)
-    profileQuery(title = "Node index scan",
+    profileQuery(title = "Node Index Scan",
                  text = """
-                          |An index scan goes through all values stored in an index, and can be used to find all nodes with a particular label having a specified property (e.g. `exists(n.prop)`).""".stripMargin,
+                          |The `NodeIndexScan` operator goes through all values stored in an index, and searches for all nodes with a particular label having a specified property (e.g. `exists(n.prop)`).""".stripMargin,
```

Comment:
"searching for" is not really correct - the index only contains nodes that match this description, so it's more accurate to say that it "returns all nodes with a particular..."

Reply:
👍